src/components/Editor/Preview/Popup.js

```diff
@@ -46,24 +53,32 @@ export class Popup extends Component<Props> {
   marker: any;
   pos: any;
 
-  componentDidMount() {
+  async componentWillMount() {
     const {
-      loadObjectProperties,
-      loadedObjects,
       value,
-      editor,
-      range
+      expression,
+      loadObjectProperties,
+      loadedObjects
     } = this.props;
```

Comment:
The function has changed to componentWillMount, so this.props will be representing the previous props. do we want to use this.props or nextProps?